Dockerfile

```
@@ -9,11 +9,10 @@ RUN --mount=type=cache,target=/root/.cache/go-build GOOS=$TARGETOS GOARCH=$TARGE
 
 FROM scratch
 
-# TODO: Configure the labels according to the target deployment
 LABEL traefik.enable=true
-LABEL traefik.http.routers.template-service.middlewares=template-service
-LABEL traefik.http.routers.template-service.rule="PathPrefix(`/api/template-service`)"
-LABEL traefik.http.middlewares.template-service.stripprefix.prefixes="/api/template-service"
+LABEL traefik.http.routers.template-service.middlewares=brain-tank-service
+LABEL traefik.http.routers.template-service.rule="PathPrefix(`/api/brain-tank-service`)"
+LABEL traefik.http.middlewares.template-service.stripprefix.prefixes="/api/brain-tank-service"
```

```suggestion
LABEL traefik.http.routers.template-service.rule="PathPrefix(`/api/brain-tank`)"
LABEL traefik.http.middlewares.template-service.stripprefix.prefixes="/api/brain-tank"
```

this matches the openapi file